build_and_push.sh

```diff
@@ -7,7 +7,8 @@ TAG="${CAIRO_VERSION}${TAG_SUFFIX}"
 docker login --username "$DOCKER_USER" --password "$DOCKER_PASS"
 
 # get requirements
-REQUIREMENTS_URL="https://raw.githubusercontent.com/starkware-libs/cairo-lang/v$CAIRO_VERSION/scripts/requirements.txt"
+# REQUIREMENTS_URL="https://raw.githubusercontent.com/starkware-libs/cairo-lang/v$CAIRO_VERSION/scripts/requirements.txt"
+REQUIREMENTS_URL="https://raw.githubusercontent.com/starkware-libs/cairo-lang/a375ecd6fbd8c82f1e87cabed9fd1544eb7bdd0d/scripts/requirements.txt"
```

Comment:
Add a comment here that this is temporary and that this refers to your fork. Perhaps also create an issue for this to be changed in the future and assign yourself (to reduce the chance of forgetting about this).

Reply:
https://github.com/0xSpaceShard/cairo-cli-docker/pull/11/commits/9b445d4e5cc7c15abafb08cae888f1a35b581722 
Issue reported [here](https://github.com/0xSpaceShard/cairo-cli-docker/issues/12)